Find a functionally equivalent code:android.graphics.drawable.NinePatchDrawable.getOpticalInsets:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -1,6 +1,6 @@
 {
     if (needsMirroring()) {
-        return Insets.of(mOpticalInsets.right, mOpticalInsets.top, mOpticalInsets.right, mOpticalInsets.bottom);
+        return Insets.of(mOpticalInsets.right, mOpticalInsets.top, mOpticalInsets.left, mOpticalInsets.bottom);
     } else {
         return mOpticalInsets;
     }

